HipercowApi/Tools/ApiErrorResponse.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Imperial College London. All rights reserved.
+
+namespace HipercowAPI.Tools
```

Comment:
Feels like this should go in "Types" or "ResponseTypes" or something rather than "Tools"